x/oracle/abci.go

```diff
@@ -49,6 +49,14 @@ func EndBlocker(ctx sdk.Context, keeper keeper.Keeper) {
 			panic(err)
 		}
 
+		ctx.EventManager().EmitEvent(
+			sdk.NewEvent(
+				types.EventTypeOracleRegistration,
+				sdk.NewAttribute(types.AttributeKeyVoteStatus, types.AttributeValueStatusCompleted),
```

Comment:
This event seems to be emitted when the vote for oracle registration is ended. If you intend to emit event of `end of vote`, I think you need to rename the value of the attribute to `AttributeValueStatusEnded` or something.

If this event emission is for passed vote, it should be moved inside the `if tallyResult.IsPassed()` statement.

Reply:
This event is emitted when voting is end. 
So I think it would be better to set the status ad `end` as @H4NLee said.

And it would be nice if the `start` event was also emitted when the voting started.